ubuntu.json

```diff
@@ -161,7 +161,12 @@
   },
   {
     "alias": "google-cloud",
-    "sourceline": "deb http://packages.cloud.google.com/apt trusty main",
+    "sourceline": "deb http://packages.cloud.google.com/apt cloud-sdk-precise main",
```

Comment:
Oh okay, see, `trusty` as an dist has gone away as an option for the Google Cloud SDK and currently doesn't work. If you'd like `google-cloud` to map to `google-cloud-trusty` and `google-cloud-precise` to map to `google-cloud-precise` then I can do that.